specification/storagecache/resource-manager/Microsoft.StorageCache/stable/2021-03-01/storagecache.json

```diff
@@ -1994,25 +1951,30 @@
         }
       }
     },
-    "UnknownTargetProperties": {
-      "description": "Storage container for use as an Unknown Storage Target.",
-      "type": "object",
-      "x-ms-discriminator-value": "unknown",
-      "allOf": [
-        {
-          "$ref": "#/definitions/StorageTargetProperties"
-        }
-      ]
-    },
     "UnknownTarget": {
+      "type": "object",
       "description": "Properties pertaining to the UnknownTarget",
       "properties": {
-        "unknownMap": {
+        "attributes": {
           "description": "Dictionary of string->string pairs containing information about the Storage Target.",
           "$ref": "#/definitions/UnknownProperties"
         }
       }
     },
+    "BlobNfsTarget": {
+      "type": "object",
+      "description": "Properties pertaining to the BlobNfsTarget.",
+      "properties": {
+        "target": {
+          "description": "Resource ID of the storage container.",
+          "$ref": "#/definitions/URLString"
+        },
+        "usageModel": {
+          "description": "Identifies the usage model to be used for this Storage Target. Get choices from .../usageModels",
```

Comment:
Description text 'Get choices from .../usageModels' is not intuitive. Recommend to re-word it as these text will be published in docs.

Reply:
Updated in latest commit.

Reply:
Also updated an existing description with a similar message.